src/api/hello.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (req, res) => {
```

Comment:
```suggestion
export default (req, res) => {
```
I would expect that to give you a 500 error though, not 404

Reply:
Oh interesting. I was following Dustin's example here: https://github.com/dmccraw/dustinmccraw/blob/main/src/api/time.ts and assumed that I would need a non ES6+ import style syntax